docker-compose.yml

```diff
@@ -119,10 +146,19 @@ services:
       - all
       - ""
 
+  vespa:
```

Comment:
@kosmikdc I'm assuming this isn't supposed to be in this branch.

Reply:
I also fixed the expose format to work with the Docker v29 release. Not sure if this is a regression in Docker (probably), but we don't use port ranges anyway, so just using the format in the standard for a single port to resolve.

https://github.com/treygrainger/ai-powered-search/commit/169f4af66671b539c6cb476c9c0ea1a19cfd04ce